File: app/src/main/java/network/devandroid/com/networklibrarycomparison/models/DataModel.java
package network.devandroid.com.networklibrarycomparison.models;

import network.devandroid.com.networklibrarycomparison.internal.NetType;

public class DataModel {

    private long mStartTime;

    private long mEndTime;

    private NetType mNetType;

    private String mUrl;

    private String mResult;

    public String getmResult() {
        return mResult;
    }

    public void setmResult(String mResult) {
        this.mResult = mResult;
    }

    public long getmStartTime() {
        return mStartTime;
    }

    public void setmStartTime(long mStartTime) {
        this.mStartTime = mStartTime;
    }

    public long getmEndTime() {
        return mEndTime;
    }

    public void setmEndTime(long mEndTime) {
        this.mEndTime = mEndTime;
    }

    public NetType getmNetType() {
        return mNetType;
    }

    public void setmNetType(NetType mNetType) {
        this.mNetType = mNetType;
    }

    public String getmUrl() {
        return mUrl;
    }

    public void setmUrl(String mUrl) {
        this.mUrl = mUrl;
    }

}

File: app/src/main/java/network/devandroid/com/networklibrarycomparison/models/DataModel.kt
package network.devandroid.com.networklibrarycomparison.models

import network.devandroid.com.networklibrarycomparison.internal.NetType

class DataModel {

    private var mStartTime: Long = 0

    private var mEndTime: Long = 0

    private var mNetType: NetType? = null

    private var mUrl: String? = null

    private var mResult: String? = null

    fun getmResult(): String? {
        return mResult
    }

    fun setmResult(mResult: String) {
        this.mResult = mResult
    }

    fun getmStartTime(): Long {
        return mStartTime
    }

    fun setmStartTime(mStartTime: Long) {
        this.mStartTime = mStartTime
    }

    fun getmEndTime(): Long {
        return mEndTime
    }

    fun setmEndTime(mEndTime: Long) {
        this.mEndTime = mEndTime
    }

    fun getmNetType(): NetType? {
        return mNetType
    }

    fun setmNetType(mNetType: NetType) {
        this.mNetType = mNetType
    }

    fun getmUrl(): String? {
        return mUrl
    }

    fun setmUrl(mUrl: String) {
        this.mUrl = mUrl
    }

}
